### DISCUSSION TITLE
Hello VVorld

### DISCUSSION THREAD

**Discussion by DebravdH** [stellar/sorobanathon#45] [General]
We decided to join in on the Soroban fun with Daddy (aka Tyler)! Below are our drawings inspired by story 1. 

Erik 3yo https://litemint.com/items/GBLP3XXL47SSAOPXQKMZ7B7NU6UDK67L4DDH6YN5TG5P3TIHJFD5HECP/Thetemplenft
Rylee 1yo https://litemint.com/items/GD2CY7EM55FT66EPD2AQSPFCYLMR7ECCNVDOZ72D4DYYD66AZXUYAJ72/Sorobanbaby
Debra (me) https://litemint.com/items/GAG2ZWVODIKOOEUELQYETGQDIQ2XUTQDQFRKUCMAHWV3KRAKIRZC4UEJ/He11oVVorld

**Comment by achallagundla** [comment]
AWWEE this is so wholesome🥺🥺😍!! Thank Erik and Rylee on our behalf for these awesome mints!

**Reply by briwylde08** [reply]
agreed I love these so much!

**Comment by rawritude** [comment]
This is such a great idea! Loving the family effort!

**Comment by stellarjohn** [comment]
I hope you have enough family members to finish the series 🤣

**Comment by stellarjohn** [comment]
Howdy DebravdH,

Congratulations!  Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 900 XLM!  One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.
Please let us know if you have any questions.

Cheers!
